### PR TITLE
Franzininho wifi ESP32-S2 Wrover

### PR DESCRIPTION
**Pull request by FBSeletronica**
Hello all!

I'm adding a port for Franzininho WiFi Board with the ESP32-S2 wrover module. I followed [this guide](https://learn.adafruit.com/how-to-add-a-new-board-to-circuitpython/customizing-the-board-files) to build the port.

Please let me know if I need to make some fixes or is everything ok.

Thanks

### CODE REVIEW

**Comment by ajs256** [comment]
`0x239A` is Adafruit's private USB Vendor ID (VID). You can get your own from USB-IF, or if your board is open-source, get one at [pid.codes](https://pid.codes). I believe @tannewt is on the team there and is hinting people in that direction.

**Comment by FBSeletronica** [comment]
> `0x239A` is Adafruit's private USB Vendor ID (VID). You can get your own from USB-IF, or if your board is open-source, get one at [pid.codes](https://pid.codes). I believe @tannewt is on the team there and is hinting people in that direction.

Sorry, I forgot to change VID in mpconfigboard.mk. Now It is ok. Thank you

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -450,7 +450,7 @@ jobs:
         - "espressif_kaluga_1"
         - "espressif_saola_1_wroom"
         - "espressif_saola_1_wrover"
-        - "franzininho_wifi_wroom"
+        - "franzininho_wifi_wrover"
```
[the remark this answers is not in the document]
Comment (FBSeletronica) [reply .github/workflows/build.yml:454]: @tannewt sorry, I made a mess with branches. Now I understand my mistake. If necessary I send a new pull request. Thank you so much for the quick reply

[the remark this answers is not in the document]
Comment (tannewt) [reply .github/workflows/build.yml:454]: No problem!

**Review by tannewt — APPROVED** [review]
Thanks! Don't forget to add your boards to https://github.com/adafruit/circuitpython-org too!

**Pull request merged**